### PR TITLE
Require Symfony 4.1.1 and remove hotfixes for 4.1.0

### PR DESCRIPTION
**Pull request by pamil**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
         "symfony/polyfill-intl-icu": "^1.3",
         "symfony/polyfill-mbstring": "^1.3",
         "symfony/swiftmailer-bundle": "^3.0",
-        "symfony/symfony": "^3.4|^4.1",
+        "symfony/symfony": "^3.4|^4.1.1",
```
Comment (lchrusciel) [inline composer.json]: Shouldn't we just conflict `4.1.0`?

Comment (pamil) [reply composer.json]: This is the same IIRC.

Comment (stefandoorn) [reply composer.json]: Does the same I think, but the difference is the 'communication'. With the conflict it's clear to everyone that some incompatibility occured.

Comment (pamil) [reply composer.json]: @stefandoorn agreed! This would require a bit of extra work, I wouldn't mind just accepting a PR that does it :)

Comment (stefandoorn) [reply composer.json]: I wouldn't mind helping on that, but that would be earliest next week somewhere. You can add an issue on my name if you want.

**Pull request merged**